views/about.lavender

```diff
@@ -38,16 +38,16 @@ body
                 a(href='#') Guide Home
               li.dropdown-header Installation
               li
-                a(href="https://github.com/golavender/lavender", title="GitHub") via Composer
+                a(href="/guide.html#composer", title="GitHub") via Composer
```

Comment:
yea leading slashes will work on gh-pages, they just don't work locally because you're editing the file directly and the leading `/` means the root of your file system instead of the website